way-cooler/README.md

```diff
@@ -0,0 +1,132 @@
+
+Currently there are very few fully-featured tiling window managers in the Wayland ecosystem, as most of the effort has been porting Gnome and KDE over. Although Wayland is still in early-stage development
+and is not backwards compatible with existing X11 tools, we wanted to put our stake in and provide for current tiling window manager users in the future.
+
+We take a lot of inspiration from current window managers (namely [i3][] and [awesome][]) but our goal is to exist as a unique alternative.
+
+
+## Current Features
+- i3-style tiling
+  * Horizontal/vertical layouts
+  * Nest containers with different layouts
+- Client application support via an IPC
+  * See an example application [here](https://github.com/Immington-Industries/Way-Cooler-Example-Clients). It displays the tree in a somewhat organized format, and is actually really helpful for both debugging the tree and understanding how subcontainers work.
+  * Enables dynamic configuration at runtime, without having to reload a configuration file
+  * Allows extensions of the window manager to exist as separate programs talking over the IPC
+- A Lua environment designed to make extending Way Cooler simple and easy
+  * Lua is the configuration format, allowing the user to enhance their window manager in any way they want.
+  * Utilities library included to aid communicating with Way Cooler
+- X programs supported through XWayland
+
+## Planned Features
+
+- i3 tabbed/stacked tiling
+- Floating windows
+- Tiling window through configurable Lua scripts (awesome-style)
+- Server-side borders around window clients
+- Swappable status bars/docs/menus
+  * A status bar built with [Conrod](https://github.com/PistonDevelopers/conrod) and Lua
+- More customization settings
+
+Follow the development of these features in our [issues section] or checkout our [contribution guidelines](#Contributing) if you want to help out.
+
+# Trying out Way Cooler
+
+If you would like to try out Way Cooler before properly installing it, then you can use the following docker command:
+```bash
+docker run --net=host --env="DISPLAY" --volume="$HOME/.Xauthority:/root/.Xauthority:rw" timidger/way-cooler
+```
+
+This allows you try out the window manager without having to install anything except Docker.
+
+# Installation
+
+## On the AUR
+
+@vinipsmaker was kind enough to provide AUR packages:
+
+[way-cooler][way-cooler-aur]
+
+[way-cooler-git][way-cooler-git-aur]
+
+## Build from source
+
+You will need the following dependencies installed on your machine to install Way Cooler:
+- Wayland
+  * Including the server and client libraries
+- wlc
+  * Installation instructions can be found on [their github page](https://github.com/Cloudef/wlc)
+- Weston (optional)
+  * The init file defaults to using `weston-terminal` as the default terminal emulator
+- Cargo
+  * The package manager / build system used by Rust
+
+Finally, to install Way Cooler simply run the following cargo command:
```

Comment:
Hm, yes I think we should change a significant portion of our README now, especially as we need to move to a different distribution model. It's totally possible for the user to compile `way-cooler-bg` and then use it in their init file, but it would be much more work, and at this point having to compile two separate programs (wlc and way-cooler-bg) just to make way-cooler work 100% is a little much. 

We can keep the manual steps, but I think first and foremost we should recommend people use our binary releases (which will have this packaged up as nicely as possible). Of course, that will be replaced by rpm/apt packages which are much easier to maintain/update.